internal/net/udp/packet_conn_test.go

```diff
@@ -0,0 +1,678 @@
+// SPDX-FileCopyrightText: 2023 The Pion community <https://pion.ly>
+// SPDX-License-Identifier: MIT
+
+//go:build !js
+// +build !js
+
+// Package udp implements DTLS specific UDP networking primitives.
+package udp
```

Comment:
Yes, we wrap the `net.PacketConn` in `fromPC` to be able to use the upstream `StressDuplex` functionality (which requires `ReadWriter` interface to be implemented). Other than that, there are a few cases where we call `ReadFrom` rather than `Read`, but the only change there is that the returned `net.Addr` is ignored.

https://github.com/pion/dtls/pull/570/files#diff-e534343f6cd79310f20cbbd44311b0173d735dc765e551d73acf5bdb3a5e6024R483 is net new to test correct routing based on identifier.